src/components/Form/Readme.mdx

```diff
@@ -97,27 +160,74 @@ Checkbox uses [rc-checkbox](https://github.com/react-component/checkbox) under t
 
 <Props of={Checkbox} />
 
+Checkbox will expose an `event.target` with
```

Comment:
Why not add these to the proptypes for `Checkbox`?

add a comment above `onChange: propTypes.func`
`/** onChange({ target: { name: <String>, type: 'checkbox', checked: <bool>} */`

or even 
`/** get's passed an object that emulated a standard JS form event, only event.target keys 'name', 'checked' and 'type' are passed.) */`

Currently the `Checkbox` still behaves non-standard.:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox

`event.target.checked` is a bool,
where as 
`event.target.value` is a string with `'on'` or `null` as its value depending on checked status. OR the value that was passed to the HTML element as an attr. If it's checked. (Didn't know that, just read up on it).

But the way we use checkboxes in fields, it expects `event.target.value` to be undefined for checkboxes...

So if you pass back `value` in the event, we'll have to fix the event handler in `Fields`

Reply:
Great point! I'll adjust